linetools/lists/linelist.py

```diff
@@ -649,18 +761,36 @@ def from_dict_to_qtable(self, a):
             tab[0][key] = a[key]
         return tab
 
+    def from_qtable_to_dict(self, tab):
+        """Converts QTable `tab` to its dictionary version.
+        An error is raised it len(tab) > 1. """
```

Comment:
I think is better to keep it here. The idea of these functions is to deal with this particular formatting of the outputs of LineList, i.e. dictionary, None, and QTables.